src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--start-day-sun">
```

Comment:
The class `calendar--start-day-sun` suggests that there might be different starting days for the calendar. Ensure that your CSS handles different classes for different start days, or this class naming could be misleading.